### DISCUSSION TITLE
Make a skin version that has the camellia mod arrows

### DISCUSSION THREAD

**Discussion by DipherL** [Saltssaumure/fnf-osu-mania-skin#18] [Ideas]
i've been wanting to see if there was a osu!mania skin of the arrows from the camellia mod, and from what i've research there is no skin that i could find that had the arrows from the mod.
so i am wondering if you could make a version of the fnf skin with the arrows from the mod, i would be grateful and glad if you could do that 

mod link in case you haven't heard about it https://gamebanana.com/mods/290601

**Comment by Saltssaumure** [comment]
These won't be added for 2 reasons:

1.   First and most important, the creator(s) of the vs Camellia mod do not give permission for others to use and distribute their assets.
2.   The arrows themselves are based on Stepmania / DDR arrows, which already exist in many other osu! skins.
